src/components/file-browser/FileBrowser.ts

```diff
@@ -22,7 +22,7 @@ interface FileBrowserProps {
   initialPath?: string;
   provider: FileBrowserProvider;
   rootLabel: string;
-  title: string;
+  title: any;
```

Comment:
Can this be any more specific than `any`?  Maybe `string | Link`?

Reply:
https://github.com/DataBiosphere/terra-ui/pull/5154/commits/1416889d5c1e5417e6d2445c2383dfd5ad825a37